core/benches/vs_s3/Cargo.toml

```diff
@@ -33,7 +33,7 @@ aws-config = { version = "0.101", features = ["behavior-version-latest"] }
 aws-credential-types = { version = "0.101", features = [
   "hardcoded-credentials",
 ] }
-aws-sdk-s3 = "0.38"
+aws-sdk-s3 = "1.4.0"
```

Comment:
There are many breaking changes in this upgrade. So you can't simply change a version.

You can run `cargo build --all-targets` to make sure they can compile correctly.

Reply:
thanks,I will try it